LayoutTests/imported/w3c/web-platform-tests/dom/idlharness-shadowrealm.window-expected.txt

```diff
@@ -1,134 +1,4 @@
-PASS AbortSignal interface: existence and properties of interface prototype object
-PASS AbortSignal interface: existence and properties of interface prototype object's "constructor" property
-PASS AbortSignal interface: existence and properties of interface prototype object's @@unscopables property
-PASS AbortSignal interface: operation abort(optional any)
-PASS AbortSignal interface: member timeout
-PASS AbortSignal interface: attribute aborted
-PASS AbortSignal interface: attribute reason
-PASS AbortSignal interface: operation throwIfAborted()
-PASS AbortSignal interface: attribute onabort
-PASS NodeList interface: existence and properties of interface object
-PASS HTMLCollection interface: existence and properties of interface object
-PASS MutationObserver interface: existence and properties of interface object
-PASS MutationRecord interface: existence and properties of interface object
-PASS Node interface: existence and properties of interface object
-PASS Document interface: existence and properties of interface object
-PASS XMLDocument interface: existence and properties of interface object
-PASS DOMImplementation interface: existence and properties of interface object
-PASS DocumentType interface: existence and properties of interface object
-PASS DocumentFragment interface: existence and properties of interface object
-PASS ShadowRoot interface: existence and properties of interface object
-PASS Element interface: existence and properties of interface object
-PASS NamedNodeMap interface: existence and properties of interface object
-PASS Attr interface: existence and properties of interface object
-PASS CharacterData interface: existence and properties of interface object
-PASS Text interface: existence and properties of interface object
-PASS CDATASection interface: existence and properties of interface object
-PASS ProcessingInstruction interface: existence and properties of interface object
-PASS Comment interface: existence and properties of interface object
-PASS AbstractRange interface: existence and properties of interface object
-PASS StaticRange interface: existence and properties of interface object
-PASS Range interface: existence and properties of interface object
-PASS NodeIterator interface: existence and properties of interface object
-PASS TreeWalker interface: existence and properties of interface object
-PASS NodeFilter interface: existence and properties of interface object
-PASS DOMTokenList interface: existence and properties of interface object
-PASS XPathResult interface: existence and properties of interface object
-PASS XPathExpression interface: existence and properties of interface object
-PASS XPathNSResolver interface: existence and properties of interface object
-PASS XPathEvaluator interface: existence and properties of interface object
-PASS XSLTProcessor interface: existence and properties of interface object
+Harness Error (FAIL), message = ReferenceError: Can't find variable: ShadowRealm
```

Comment:
What's happening here? Seems we lost a LOT of test coverage so not great.

Reply:
Oh my bad, I'll exclude this file. It's annoyingly generated for dom/ and url/ and some other directories when you do --reset-results. Given that EWS isn't complaining I kinda suspect it just needs to be cleaned up, but not as part of this PR.

Reply:
It's exactly as I feared:
```
LayoutTests/TestExpectations:imported/w3c/web-platform-tests/compression/idlharness-shadowrealm.window.html [ Pass Failure ]
LayoutTests/TestExpectations:imported/w3c/web-platform-tests/dom/idlharness-shadowrealm.window.html [ Pass Failure ]
LayoutTests/TestExpectations:imported/w3c/web-platform-tests/encoding/idlharness-shadowrealm.window.html [ Pass Failure ]
LayoutTests/TestExpectations:imported/w3c/web-platform-tests/hr-time/idlharness-shadowrealm.window.html [ Pass Failure ]
LayoutTests/TestExpectations:imported/w3c/web-platform-tests/html/dom/idlharness-shadowrealm.window.html [ Pass Failure ]
LayoutTests/TestExpectations:imported/w3c/web-platform-tests/performance-timeline/idlharness-shadowrealm.window.html [ Pass Failure ]
LayoutTests/TestExpectations:imported/w3c/web-platform-tests/streams/idlharness-shadowrealm.window.html [ Pass Failure ]
LayoutTests/TestExpectations:imported/w3c/web-platform-tests/url/idlharness-shadowrealm.window.html [ Pass Failure ]
LayoutTests/TestExpectations:imported/w3c/web-platform-tests/user-timing/idlharness-shadowrealm.window.html [ Pass Failure ]
```
Maybe I'll take some time to clean this up as I keep running into this nonsense.